mod.json

```diff
@@ -2,10 +2,10 @@
   "_QPVersion": "0.1.1",
   "id": "BeatLeader",
   "name": "BeatLeader",
-  "version": "0.1.1",
+  "version": "1.0.1",
   "author": "NSGolova",
   "packageId": "com.beatgames.beatsaber",
-  "packageVersion": "1.19.1",
+  "packageVersion": "1.20.0",
   "coverImage": "cover.png",
   "modFiles": [
     "libbl.so"
```

Comment:
Need to add bshook lib to the `libraryFiles` because core mods were not copying the right version by default for the 1.20

Reply:
My bad